src/plugins/discover/public/application/apps/main/components/view_mode_toggle/constants.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const DISCOVER_VIEW_MODES = {
```

Comment:
:art: Could we make this an `enum` instead?

Reply:
Updated to enum here https://github.com/elastic/kibana/pull/107184/commits/f12711a802b6208952ba700785aa83a6cdddb615